docs/tech-reference/yum-plugins.rst

```diff
@@ -627,3 +627,13 @@ Optional Configuration Parameters
 ``force_full``
 Boolean flag to indicate whether or not publish should be done from scratch.
 If unspecified the incremental publish will be performed when possible.
+
+``remove_old_repodata``
+Boolean flag to indicate whether or not repodata passed expiration are removed
+during publish. Files that present in repomd.xml are preserved no matter of 
+their age.
+
+``remove_old_repodata_threshold``
+If ``remove_old_repodata`` is true, this attribute specify maximal age of 
+repodata in seconds that are not removed during publish. Hard coded to 2 weeks.
```

Comment:
I also suggest sticking to the units that we expect it to be expressed it. In case the default is too cumbersome to express in seconds, then I'd change the units to something that better matches the default. Otherwise express the default in seconds.